src/inspection/src/inspection_functions.cpp

```diff
@@ -0,0 +1,43 @@
+#include "include/inspection_functions.hpp"
+
+InspectionFunctions::InspectionFunctions() {define_constants("src/inspection/src/config.txt");}
+
+void InspectionFunctions::define_constants(const std::string &path) {
```

Comment:
This would be beter suited to be called (fromFile) or something like that, leaving you the alternative of using the constructor to define the constants without being from file, which is better suited for tests.